packages/@o3r/core/schematics/component/container/schema.json

```diff
@@ -41,8 +41,8 @@
     "useComponentFixtures": {
       "type": "boolean",
       "description": "Generate component fixtures for tests",
-      "default": true,
-      "x-prompt": "Generate component fixtures for tests?"
+      "default": null,
+      "nullable": true
```

Comment:
Is it supported by `nx` runner?

Reply:
As discussed NX support will be fixed in a separate PR